README.rst

```diff
@@ -176,17 +176,16 @@ We can now install the REANA command-line client, run the analysis and download
     $ export REANA_SERVER_URL=https://reana.cern.ch/
     $ export REANA_ACCESS_TOKEN=XXXXXXX
     $ # create a new workflow
-    $ reana-client create -n my-analysis
-    $ export REANA_WORKON=my-analysis
+    $ reana-client create -n myanalysis
+    $ export REANA_WORKON=myanalysis
     $ # upload input code and data to the workspace
```

Comment:
Shall we keep "upload input code and data to the workspace"?  Now that the workflow specs are also uploaded...

Option A. Keep it, because the language is very researcher-friendly.

Option B. Amend to something more precise like "upload input code, data and workflow to the workspace".

Option C. Amend to something neutral-sounding like "upload workflow inputs into the workspace".

Option D. Amend to even more generic language like "populate workspace".

Reply:
We should take into account that in the future we will upload the entire directory, so I'm more inclined to generic option D).

Reply:
D is nice, but the verbal connection between "populate" and `upload` is a bit lost... "upload myanalysis to the workspace"?

Reply:
BTW one more thing: since people were confused a bit before by "myreana" venv name, perhaps we can  also change the leading part of the console to something like:

```
    $ # install reana-client
    $ virtualenv ~/.virtualenvs/reana
    $ source ~/.virtualenvs/reana/bin/activate
    $ pip install reana-client
    # connect to some REANA cloud instance
    ...
```

Reply:
FWIW I guess I like B the most, since it is an explicit help over implicit statement, which is good for user docs and knowledge sharing. Moreover, the expression mirrors the three pillars that we have (=data, code, workflow) with the fourth pillar (=environment) being docker-pulled during runtime, so there is nothing to upload here.  I think the analogy works nicely. (Although the text B may be perhaps a bit long...)

Reply:
Changed. Take a look to see how it looks.